### ISSUE TITLE
Bring back audio player

### ISSUE THREAD

**Issue by kfandrew910** [joel16/3DShell#82]
In version 4.3.1 there used to be an audio player, now in current version (v5.1.0) I can't play mp3 files. 

Please bring this feature back

**Comment by yhqe** [comment]
we need this, the 4.3.1 audio player is so broken you can't even play any audio with cover art
there aren't any other audio players that are actually good for the 3ds, this *was* the only one

**Comment by nycki93** [comment]
Agreed! The audio player in 4.3.1 is so good it puts the official app to shame. Please keep it in!

**Comment by joel16** [comment]
I was working on a re-write a while back but I don't have any ETAs at the moment. However I'll keep this in mind.

**Comment by mybug31** [comment]
This is going to sound crazy, but is it possible to listen to mp3 files while viewing jpgs? or let music continue to play while we browse files?

**Comment by mememachine5050** [comment]
i would fucking *love* to get mp3 support back, and maybe hopefully album artwork too :)

**Comment by dexx0008** [comment]
In addition to bringing back music player, it'd be nice to keep music player running without needing to plug in headphones. 

perhaps by utilizing this code. https://github.com/LumaTeam/Luma3DS/blob/master/sysmodules/rosalina/source/minisoc.c#L58-L91

**Comment by sqcrtx** [comment]
I agree, the player must be resurrected.

**Comment by TheeSplatt** [comment]
PLEASE this would be amazing!!!! i would love to listen to music on my 3ds wherever i go but none are as good as 3dshell was 3:

**Comment by Rover-N** [comment]
I just wanted to say there is still an audience for this feature!

**Comment by joel16** [comment]
I really appreciate the support, although I haven't really posted anything. I can confim audio support will be back in the re-write as I have an early test screenshot:

<img width="240" height="800" alt="Image" src="https://github.com/user-attachments/assets/cd797f53-8a57-4cd9-b695-6923a4a66c51" />

This new version will also introduce a bonus feature that was not avaialble in the previous 3DShell releases.

**Comment by joel16** [comment]
Closing with https://github.com/joel16/3DShell/releases/tag/next-v1.00
